docs/README.md

```diff
@@ -29,7 +35,11 @@ Under construction
 
 ### Linux
 
-Under construction
+1. Install [Docker](https://docs.docker.com/engine/installation/)
+2. Build Docker image. From the rippled root folder:
+```
+docker build -t rippled-docs docs/
```

Comment:
I followed the instructions to install docker. If I run the command above I get the following message: `Cannot connect to the Docker daemon. Is the docker daemon running on this host?`. However, if I use `sudo` the above appears to work (it's still running). Is this expected? If so, we should document that `sudo` is required. If not we should understand what I'm doing wrong.